docs/pages/access-controls/teleport-policy/policy-connections.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Teleport Policy Connections
+description: Connections in Access Graph with Teleport Policy.
+---
+
```

Comment:
I would add a brief intro paragraph here to indicate what this page is about and when a user would want to read it.